.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,32 @@
+name: Update NPM
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - run: yarn install --frozen-lockfile
+      - run: yarn format
+      - run: yarn test:unit
+      - run: yarn test:int
```

Comment:
Same comment as above, I think in this environment we just want to use `yarn test`